products/opensuse/main.pm

```diff
@@ -391,6 +391,7 @@ sub load_consoletests() {
             loadtest "console/http_srv.pm";
             loadtest "console/mysql_srv.pm";
             loadtest "console/dns_srv.pm";
+            loadtest "console/openvswitch.pm";
             if (check_var('ARCH', 'x86_64')) {
                 loadtest "console/docker.pm";
```

Comment:
As it's on screen here: the docker test should move too - but that's of course not your task (just mentioning it to the others on CC)